src/main/scala/com/codacy/pmd/PMD.scala

```diff
@@ -40,7 +40,10 @@ object PMD extends Tool {
           .filter(filename => !Languages.invalidExtensions.exists(filename.endsWith))
           .mkString(",")
     }
-    pmdConfig.setInputPaths(filesStr)
+
+    if (filesStr.nonEmpty) {
+      pmdConfig.setInputPaths(filesStr)
```

Comment:
What happens when there are no input paths specified? Will it skip the analysis or analyse something else?

Reply:
It will not analyse any files.
I tried to reproduce this on the multiple tests by adding a `sample.java` (with issues) and 2 other files that are on the invalid extensions list. The java file is on the ignored files (`BeforeExecutionExclusionFileFilter`) so it is not sent by Codacy to be analysed (@lolgab correct me if this is not the expected behaviour). This way, the filesStr will be empty and `setInputPaths` will not be called. The expected behaviour is to not analyse neither the filtered files, neither the java one, so 0 results should be returned.